src/explore-education-statistics-admin/src/pages/methodology/edit-methodology/status/MethodologyStatusPage.tsx

```diff
@@ -61,8 +61,14 @@ const MethodologyStatusPage = ({
     const nextSummary = await methodologyService.updateMethodology(
       methodologyId,
       {
-        ...model.summary,
-        ...values,
+        latestInternalReleaseNote: values.latestInternalReleaseNote,
```

Comment:
Not sure if we do this elsewhere but it might be worth destructuring these values to make this a little less verbose:   
`const { publishingStrategy, status, withReleaseId, latestInternalReleaseNote }  = values`.

Reply:
Done!